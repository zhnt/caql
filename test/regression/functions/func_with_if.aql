function check_number() {
    let x = 5
    if x > 3 {
        print("Number is greater than 3")
    } else {
        print("Number is not greater than 3")
    }
}

check_number()
