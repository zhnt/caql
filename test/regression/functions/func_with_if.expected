Number is greater than 3

